app/src/main/java/app/attestation/auditor/attestation/AttestationPackageInfo.java

```diff
@@ -61,8 +61,16 @@ public long getVersion() {
 
     @Override
     public String toString() {
-        return "Package name: " + getPackageName() +
```

Comment:
What I mean is that for example here, I think it will probably compile to equally good code either way.